src/app/workflow/workflow.js

```diff
@@ -240,13 +245,14 @@ angular.module('app.workflow', [
 
 .controller('WorkflowStepFormController', function($scope, $state, $stateParams, mediator, workflow, workflowManager, forms) {
   var self = this;
-
   self.workflow = workflow;
   self.forms = forms;
   self.step = workflow.steps.filter(function(item) {
     return item.code === $stateParams.code;
   })[0];
   mediator.subscribeForScope('wfm:workflow:updated', $scope, function(workflow) {
+    //remove generated angular variables.
+    workflow = JSON.parse(angular.toJson(workflow));
```

Comment:
Same as above, this topic is coming from the raincatcher-workflow module. It would be better to change it there.

https://github.com/feedhenry-raincatcher/raincatcher-workflow/blob/master/lib/angular/directive.js#L144